CFC_WebApp/dao/socialmediauser.py

```diff
@@ -0,0 +1,235 @@
+"""
+Use this file to add all functions related to definition of socialmedia user
+without polluting main api (as much as possible).
+"""
+import json
+from urllib import urlencode
+import facebook
+from dao.user import User
+from get_database import get_uuid_db
+
+# Seems more appropriate to have these here, but can move to the ../socialmedia.py file or to the main api
+fb_config = open('clients/socialmedia/settings.json')
+fb_data = json.load(fb_config)['client_settings']
+FACEBOOK_APP_ID = fb_data['fb_app_id']
+FACEBOOK_APP_SECRET = fb_data['fb_app_secret']
+
+
+class FacebookUser:
```

Comment:
Yes. That is why this is all in a file called SocialmediaUser. I have not stubbed these out yet, but I did consider that we will want additional services as an option.
Sid Feygin
sid.feygin@gmail.com



> On Jun 2, 2015, at 11:29 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/dao/socialmediauser.py <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31596787>:
> 
> > +without polluting main api (as much as possible).
> > +"""
> > +import json
> > +from urllib import urlencode
> > +import facebook
> > +from dao.user import User
> > +from get_database import get_uuid_db
> > +
> > +# Seems more appropriate to have these here, but can move to the ../socialmedia.py file or to the main api
> > +fb_config = open('clients/socialmedia/settings.json')
> > +fb_data = json.load(fb_config)['client_settings']
> > +FACEBOOK_APP_ID = fb_data['fb_app_id']
> > +FACEBOOK_APP_SECRET = fb_data['fb_app_secret']
> > +
> > +
> > +class FacebookUser:
> So is the plan that FacebookUser is an aspect of a regular user. Because we might want to have TwitterUser, WhatsappUser etc, all defined for the same user, if the user has logged into multiple external services at the same time.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31596787>.
>